app/src/main/java/org/p2p/wallet/settings/SettingsModule.kt

```diff
@@ -1,67 +1,35 @@
 package org.p2p.wallet.settings
 
-import org.koin.android.ext.koin.androidContext
 import org.koin.core.module.dsl.factoryOf
 import org.koin.dsl.bind
 import org.koin.dsl.module
 import org.p2p.wallet.common.di.InjectionModule
 import org.p2p.wallet.settings.interactor.SettingsInteractor
 import org.p2p.wallet.settings.interactor.ThemeInteractor
 import org.p2p.wallet.settings.model.SettingsItemMapper
-import org.p2p.wallet.settings.ui.appearance.AppearanceContract
-import org.p2p.wallet.settings.ui.appearance.AppearancePresenter
 import org.p2p.wallet.settings.ui.network.SettingsNetworkContract
 import org.p2p.wallet.settings.ui.network.SettingsNetworkPresenter
-import org.p2p.wallet.settings.ui.newreset.pin.NewResetPinContract
-import org.p2p.wallet.settings.ui.newreset.pin.NewResetPinPresenter
 import org.p2p.wallet.settings.ui.recovery.RecoveryKitContract
 import org.p2p.wallet.settings.ui.recovery.RecoveryKitPresenter
-import org.p2p.wallet.settings.ui.reset.ResetPinContract
-import org.p2p.wallet.settings.ui.reset.ResetPinPresenter
-import org.p2p.wallet.settings.ui.reset.seedphrase.ResetSeedPhraseContract
-import org.p2p.wallet.settings.ui.reset.seedphrase.ResetSeedPhrasePresenter
-import org.p2p.wallet.settings.ui.security.SecurityContract
-import org.p2p.wallet.settings.ui.security.SecurityPresenter
-import org.p2p.wallet.settings.ui.settings.NewSettingsContract
-import org.p2p.wallet.settings.ui.settings.NewSettingsPresenter
-import org.p2p.wallet.settings.ui.zerobalances.SettingsZeroBalanceContract
-import org.p2p.wallet.settings.ui.zerobalances.SettingsZeroBalancesPresenter
+import org.p2p.wallet.settings.ui.resetpin.pin.ResetPinContract
+import org.p2p.wallet.settings.ui.resetpin.pin.ResetPinPresenter
+import org.p2p.wallet.settings.ui.settings.SettingsContract
+import org.p2p.wallet.settings.ui.settings.SettingsPresenter
+import org.p2p.wallet.settings.ui.settings.SettingsPresenterAnalytics
 
 object SettingsModule : InjectionModule {
 
     override fun create() = module {
         factoryOf(::SettingsInteractor)
         factoryOf(::ThemeInteractor)
+
         factoryOf(::SettingsItemMapper)
-        factory {
-            // holy shit, TODO smth with this dependencies!
```

Comment:
did something with with these dependencies :))

Reply:
that was easy :D